.github/workflows/integration-tests.yml

```
@@ -21,7 +21,7 @@ jobs:
       working-directory: ./openverse-api
       run: |
         pip install --upgrade pipenv
-        pipenv install --dev
+        pipenv install pre-commit
         pipenv run pre-commit run --all-files
```

Since switching to `pipenv install pre-commit` did not speed things up, this might:
```suggestion
        pip install pre-commit
        pre-commit run --all-files
```